### PR TITLE
Add color knob

### PR DESCRIPTION
**Pull request by elisherer**
color-type-knob

Add a color picker.
This is a presentational plugin and should support a basic presentation switch which is the color.
now supported by all browsers, and until now was used in a select instead of using the native color picker.

Based on 'number', and as such, was inserted right after it in all the code appearances.

### CODE REVIEW

**Comment by arunoda** [comment]
Seems like input color has some good coverage: https://caniuse.com/#feat=input-color

I will try how looks like in different browsers and take this in.

**Comment by elisherer** [comment]
I have thought of this and compared its coverage with input date, which has less coverage  (http://caniuse.com/#search=input-date) and has a knob.
I think it should be the concern of the developer, maybe add a disclaimer under each of the knobs saying support is browser dependant.

**Comment by arunoda** [comment]
Seems like it's not working with Safari and I don't like the color window.
I prefer to use something like this: https://github.com/casesandberg/react-color

Which is pretty decent and simple to use.

**Comment by elisherer** [comment]
Changed it to react-color

File: example/stories/index.js [hunk 1]
```diff
@@ -25,11 +26,12 @@ stories.add('with all knobs', () => {
   const dob = date('DOB', new Date('January 20 1887'));
 
   const bold = boolean('Bold', false);
-  const color = select('Color', {
-    red: 'Red',
-    green: 'Green',
-    black: 'Black'
-  }, 'black');
+  const textColor = color('Color', 'black');
+  const textDecoration = select('Decoration', {
```
Comment (arunoda) [inline example/stories/index.js]: Remove this. This seems like not related to this PR.

**Comment by elisherer** [comment]
colorswitch2
colorswitch
Done and Done.. :)

**Comment by arunoda** [comment]
I took this with some minor changes and here it is: https://git.io/vXdhZ

**Comment by arunoda** [comment]
Released with `v1.6.0`
